seaice/pipeline/slurm.py

```diff
@@ -0,0 +1,235 @@
+"""Helper functions for writing Slurm scripts for data processing. Functions should get called from the pipeline processing notebook. Each Slurm function calls out to a Python script."""
```

Comment:
Do we need Slurm for this scale of processing?

Reply:
Strictly, no! But this does re-use our existing template and adds some performance. And it is a "learning vehicle" for me 😀